src/dropArgs.js

```diff
@@ -0,0 +1,23 @@
+import { nAry } from 'ramda';
+
+/**
+ * Accepts a function with any arity and returns a function with arity of zero.
+ * The function returned ignores any arguments supplied to it.
+ *
+ * @func dropArgs
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.10.0|v2.10.0}
+ * @category Logic
+ * @sig (...a -> b)-> () -> b
+ * @param {Function} fn The function with any arity
+ * @return {Function} Returns function with arity of zero
+ * @see {@link http://ramdajs.com/docs/#nAry|R.nAry}
+ * @example
+ *
+ * const fn = (a = 1, b = 2) => a + b;
+ *
+ * RA.dropArgs(fn)(3, 4); //=> 3
```

Comment:
RA.dropArgs(fn)("ignored1", "ignored2");

Reply:
@char0n you missed it here

Reply:
Thanks caught it here: https://github.com/char0n/ramda-adjunct/commit/d4c2add6a63787570547a02c16c5e1da0422a228